dali/operators/reader/loader/video_loader.cc

```diff
@@ -176,8 +178,16 @@ std::vector<dali::file_meta> filesystem::get_file_label_pair(
     s.close();
   } else {
     file_info.reserve(filenames.size());
-    for (const auto & f : filenames) {
-      file_info.push_back(file_meta{f, 0, 0, 0});
+    for (size_t i = 0; i < filenames.size(); ++i) {
+      int label = 0;
+      if (use_labels) {
+        if (labels.size()) {
+          label = labels[i];
```

Comment:
Is the size of `labels` validated anywhere? I mean that it's not shorter than `filenames.size()`.

Reply:
Ok, I see it in video_reader_op.h

Reply:
That is correct